guides/common/modules/proc_synchronizing-ansible-collections.adoc

```diff
@@ -49,7 +49,7 @@ endif::[]
 .. To sync {Project} from `console.redhat.com`, enter your token in the *Auth Token* field and enter your SSO URL in the the *Auth URL* field.
 ifdef::satellite[]
 +
-For more information, see https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/1.2/html-single/managing_red_hat_certified_and_ansible_galaxy_collections_in_automation_hub/index#proc-create-api-token[Retrieving your Red{nbsp}Hat Certified Collections Sync URL and API token] in the _Managing Red{nbsp}Hat Certified and Ansible Galaxy collections in Automation Hub_ guide.
+For more information, see https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.3/html-single/getting_started_with_automation_hub/proc-create-api-token[Retrieving your Red{nbsp}Hat Certified Collections Sync URL and API token] in the _Managing Red{nbsp}Hat Certified and Ansible Galaxy collections in Automation Hub_ guide.
```

Comment:
```suggestion
For more information, see https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.3/html-single/getting_started_with_automation_hub/index#proc-create-api-token[Retrieving your Red{nbsp}Hat Certified Collections Sync URL and API token] in the _Managing Red{nbsp}Hat Certified and Ansible Galaxy collections in Automation Hub_ guide.
```

Like this @adamlazik1 ?

Reply:
Yes, that works, thanks!

Reply:
Side note: I'll have a look at your opened PRs later today.

Reply:
good question. I have no strong preferences. I wonder if we should be in sync with linking internal guides which we link as single-page, but honestly I am fine with both.

Reply:
Then, I would prefer Multi-Page considering the DDF advantage for end-users. We recommend Multi-Page over Single for customer-facing docs.